sitemap

```diff
@@ -0,0 +1,48 @@
+Landing Page
```

Comment:
This is pretty cool~ Is there documentation we could link in a note somewhere that explains how this was generated?

Reply:
Absolutely! I can prepare a blurb about how it's generated.